includes/admin/core/class-admin-builder.php

```diff
@@ -655,6 +655,7 @@ public function update_field() {
 						}
 					} elseif ( false !== strpos( $key, 'um_editor' ) ) {
 						if ( 'block' === $array['post']['_type'] ) {
+							$val                          = str_replace( "\r\n\r\n", '<br>', $val );
```

Comment:
maybe the `nl2br()` function is the better solution?
https://www.php.net/manual/ru/function.nl2br.php